z2/resources/api_macros.tera.md

```diff
@@ -58,6 +58,12 @@ The block number can be:
 
 If the block number is not provided, `latest` is assumed.
 
+# blockid
+
+### Block ID
+
+A block identifier, as specified by https://eips.ethereum.org/EIPS/eip-1898.
```

Comment:
cc @rrw-zilliqa - Do you think its acceptable to just delegate to the EIP here?

Reply:
I don't see why not, assuming you aim to support the whole EIP. Mildly harder to look up, but that's about the only factor I can even think of ..